kernel/src/callback.rs

```diff
@@ -1,7 +1,7 @@
 use core::nonzero::NonZero;
 use process;
 
-#[derive(Clone,Copy)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
This is an example of a struct where we'll actually want a more subtle implementation of Debug since it'll have to contain "secret" info (like a liveness nonce). but, for now fine